.eslintrc

```diff
@@ -6,6 +6,13 @@
   "rules": {
     "func-names": "off",
     "no-console": "off",
-    "prefer-template": "off"
+    "prefer-template": "off",
+    "node/no-unpublished-require": "off"
```

Comment:
Maybe we should create a `.eslintrc` in the server tests directory and make it extend from `server/.eslintrc`. Rules like this make no sense in tests, but may be useful in the main code.

Reply:
:+1: